zingolib/src/wallet/traits.rs

```diff
@@ -257,18 +265,25 @@ impl Recipient for zcash_primitives::sapling::PaymentAddress {
     }
 }
 
-pub trait CompactOutput<D: DomainWalletExt>:
-    Sized + ShieldedOutput<D, COMPACT_NOTE_SIZE> + Clone
+fn slice_to_array<const N: usize>(slice: &[u8]) -> &[u8; N] {
+    <&[u8; N]>::try_from(slice).unwrap_or(&[0; N])
+    //todo: This default feels dangerous. Find better solution
```

Comment:
dangerous enough to not merge into dev yet??

Reply:
This is already in dev, this is a move https://github.com/zingolabs/zingolib/pull/743/files#diff-85a6f01ada5ff7e4b57dcd0d86ded00dd41c6f7f2e05b0367fc2e2c84d3c3cd9L111-L114 and not new logic.